src/_pytest/assertion/util.py

```diff
@@ -436,16 +506,21 @@ def _compare_eq_cls(
     if same and verbose < 2:
         explanation.append("Omitting %s identical items, use -vv to show" % len(same))
     elif same:
-        explanation += ["Matching attributes:"]
-        explanation += pprint.pformat(same).splitlines()
+        explanation += ["{}Matching attributes:".format(indentation)]
+        mystr = [indentation + pprint.pformat(same).splitlines()[0]]
```

Comment:
The code changed here -- instead of including all of the lines of `pprint.pformat(same).splitlines()`, it now only includes the first.

Also, it would be better to avoid assigning to `mystr` here, and include it in the line below directly, because `mystr` is not used again.